components/style/values/computed/pointing.rs

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Computed values for Pointing properties.
+//!
+//! https://drafts.csswg.org/css-ui/#pointing-keyboard
+
+#[cfg(feature = "gecko")]
+use std::fmt;
+#[cfg(feature = "gecko")]
+use style_traits::ToCss;
+use style_traits::cursor::CursorKind;
+#[cfg(feature = "gecko")]
+use values::specified::url::SpecifiedUrl;
+
+/// The computed value for the `cursor` property.
+///
+/// https://drafts.csswg.org/css-ui/#cursor
+#[cfg(feature = "servo")]
+#[derive(Clone, Copy, Debug, MallocSizeOf, PartialEq, ToComputedValue, ToCss)]
+pub struct Cursor(pub CursorKind);
+
+/// The computed value for the `cursor` property.
+/// Gecko variant.
+/// https://drafts.csswg.org/css-ui/#cursor
+#[cfg(feature = "gecko")]
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue)]
+pub struct Cursor {
+    /// The parsed images for the cursor.
+    pub images: Vec<CursorImage>,
```

Comment:
We should also file followups for:

 * Converting `images` to `Box<[CursorImage]>`.
 * Derive `ToCss` (I think it can be done now using `#[css(comma)]` on the `images` field.

Would you mind filing those issues? I can do that myself if not otherwise.